### ISSUE TITLE
Home Page A11y improvements

### ISSUE THREAD

**Issue by scottbenton** [scottbenton/Iron-Journal#263]
1. Provide a "Skip to Main Content" link. The first focusable element of the page should be a link offering to move the user to the unique page content (past the banner landmark or repeated-across-pages content). This element can be hidden until it receives keyboard focus.
2. When activating the Consult the Oracle button, there is no screen reader feedback that anything has happened.
3. The stat buttons (5 core stats + supply) are a button element with a p child. The p child causes NVDA to read out the button structure twice: "Button Edge. Button +2."
4. Supply Track buttons provide no feedback on activation.
5. To consider: is group the appropriate role for the supply track? Possibly yes, but need to review https://www.w3.org/WAI/ARIA/apg/ to consider other possibilities.
6. Progress tracker doesn't supply feedback when marking/removing progress.
7. Both focus indicators fail contrast requirements. The "bubble" format doesn't fully encompass the elements in question and fails contrast against the dark backgrounds of the heading (2.37 color contrast) The "pink ring" keyboard focus style looks like a 1.01 contrast ratio against bright backgrounds. Requires at least 3.)
8. The roll log in general is problematic, but that can come later.
9. I suspect the login and registration options are a third-party tool and beyond your capacity to fix, but those have some quirks too.
10. I see you have <section>s with an aria-labelledby, but I would advise you actually de-label these: sections tend to get passed on quietly in screen readers unless they have labels, and the label of these sections are implied by the heading elements. It's causing a bit of double-reading from NVDA.

**Comment by scottbenton** [comment]
1. Done- a skip content link has been added.
2. Done - an aria-live region now announces the change to the user
3. Done - changed p to span
4. Blocked: Need to ask entitas if a slider would be a good method for this and for the progress tracks https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-rating/
5. Blocked: Same as above
6. Blocked: Same as above
7. ToDo
8. Done, needs feedback: Roll log still exists, but now rolls are announced via an aria-live region
9. Will be another ticket
10. Done

**Comment by scottbenton** [comment]
Update: created #265 for track -> slider conversions, and #266 for the focus indicator contrast issue.
